arithmetization/src/main/java/net/consensys/linea/zktracer/opcode/OpCodes.java

```diff
@@ -32,27 +33,29 @@
 public class OpCodes {
   private static final JsonConverter YAML_CONVERTER = JsonConverter.builder().enableYaml().build();
 
-  private static Map<Integer, OpCodeData> valueToOpCodeDataMap;
-  private static Map<OpCode, OpCodeData> opCodeToOpCodeDataMap;
+  private static Map<Integer, OpCodeData> valueToOpCodeDataMap = new ConcurrentHashMap<>();
+  private static Map<OpCode, OpCodeData> opCodeToOpCodeDataMap = new ConcurrentHashMap<>();
```

Comment:
these vars are overwritten with regular maps in the load method, probably not what you want, and as Florian said below, they could also be loaded at class initialization time, without the need to `OpCodes.load` in `ZkTracer`

Reply:
I changed to a static initialization, no need for the concurrent hashmaps here.